pants.toml

```
@@ -34,6 +34,7 @@ backend_packages.add = [
   "pants.backend.experimental.scala.debug_goals",
   "pants.backend.experimental.visibility",
   "pants.backend.tools.preamble",
+  "pants.backend.tools.workunit_logger",
```

Should this be scoped as `experimental` or `debugging` similar? For instance, I could imagine we don't want to commit to the adhoc output format (and approach to saving to disk), versus potentially outputting a format like OpenTelemetry traces or Zipkin or similar.

```suggestion
  "pants.backend.experimental.tools.workunit_logger",
```

---

What about `raw_workunit_logger` and we don't filter the object keys?

---

Are you suggesting not making this experimental? I think anything that is exposing the "raw" work unit structure should be experimental, or else changes to the pants internals might need to be considered a user API change, rather than "just" a plug-in API change. 

That is, if this is experimental, it seems like it gives us scope to merge this easily without having to nail down the format, whereas being stabilised immediately would presumably require more thought to perfect it.

---

Yea, +1 on experimental. No other changes needed in that case I think.